templates/compose/palworld.yaml

```
@@ -0,0 +1,131 @@
+version: '3.7'
+services:
+  palworld:
+    image: 'thijsvanloef/palworld-server-docker:latest'
+    restart: unless-stopped
+    container_name: palworld-server
+    stop_grace_period: 30s
+    ports:
+      - '8211:8211/udp'
+      - '27015:27015/udp'
+    volumes:
+      - '${COOLIFY_VOLUME_APP}:/palworld/'
```

Use a name for volume!!
```suggestion
      - 'palword-data:/palworld/'
```